neural_compressor/torch/algorithms/weight_only/awq.py

```diff
@@ -0,0 +1,627 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Please use `2024` for newly added files. Please check other files as well.

Reply:
fixed https://github.com/intel/neural-compressor/pull/1561/commits/79021b4e2f667bbe77e353dfe195c5d29a9d668a